### PR TITLE
initial commit side filters

### PR DESCRIPTION
**Pull request by snathanvj**
#95
ebi-ait/data-catalogue#95

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,11 @@
     "react-router-dom": "^6.22.0",
     "react-scripts": "5.0.1",
     "typescript": "^4.9.5",
-    "web-vitals": "^2.1.4"
+    "web-vitals": "^2.1.4",
+    "@mui/icons-material": "latest",
+    "@mui/material": "latest",
+    "@mui/x-data-grid": "latest",
```
Comment (amnonkhen) [inline package.json:26]: This needs to be deleted, as we are not using mui's data-grid.

File: src/SideFilter.jsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                mb={2}
+            >
+                <Box fontSize="h6.fontSize" fontWeight="bold">
+                    Filters
+                </Box>
+                <Button variant="text" color="primary" onClick={handleResetAll}>
+                    Reset All
+                </Button>
+            </Box>
+            <List>
```
Comment (amnonkhen) [inline src/SideFilter.jsx:72]: I beieve we should use [the auto-complete](https://mui.com/material-ui/react-autocomplete) control here. Start simple, only allowing a single selection, and we'll work our way up.

**Review by amnonkhen — CHANGES_REQUESTED** [review]

**Comment by amnonkhen** [comment]
please make sure to connect the pr to the ticket. It makes review easier and quicker.

File: src/SideFilter.jsx
Comment (theisuru) [inline src/SideFilter.jsx:1]: We should use `TypeScript` instead of `JavaScript`

Comment (theisuru) [reply src/SideFilter.jsx:1]: Check the [multi select section](https://mui.com/material-ui/react-select/). Use `TS` box to see TypeScript code.

**Pull request merged**